.github/scripts/test.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```

Comment:
It'd be great to have a high-level explanation what we are doing here. E.g., we are unpacking sdists and trying to run HLS on it? Why? Is that something from GitLab I just didn't know about?